onnxruntime/core/providers/stvm/stvm_execution_provider.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef STVM_EXECUTION_PROVIDER_H
+#define STVM_EXECUTION_PROVIDER_H
+
+#include <string>
+#include <vector>
+#include <memory>
+#include <unordered_map>
+
+#include "core/common/logging/logging.h"
+#include "core/framework/execution_provider.h"
+#include "core/providers/stvm/stvm_execution_provider_info.h"
+#include "core/platform/ort_mutex.h"
+
+#include "stvm_common.h"
+
+namespace onnxruntime {
+
+namespace stvm_env_vars {
+   static const std::string kDumpSubgraphs = "ORT_STVM_DUMP_SUBGRAPHS";
+}  // namespace stvm_env_vars
+
+class STVMRunner;
+
+// Logical device representation.
```

Comment:
Improve the comment here, doesn't seem to make sense with respect to the execution provider.

Reply:
removed